app/src/main/java/com/testfairy/TestFairyFeedbackOverlay.java

```diff
@@ -0,0 +1,614 @@
+/*
+ * Copyright (C) 2019 TestFairy
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Icons are available under Apache License, Version 2.0 (the "License")
+ * and provided by material.io project.
+ */
+
+package com.testfairy;
+
+import android.app.Activity;
+import android.app.Application;
+import android.content.Context;
+import android.content.res.Resources;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+import android.graphics.RectF;
+import android.graphics.Shader;
+import android.graphics.drawable.BitmapDrawable;
+import android.os.Bundle;
+import android.util.AttributeSet;
+import android.util.Base64;
+import android.util.Log;
+import android.util.TypedValue;
+import android.view.Gravity;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.view.animation.Animation;
+import android.view.animation.TranslateAnimation;
+import android.widget.FrameLayout;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.Toast;
+
+import com.testfairy.activities.ProvideFeedbackActivity;
+
+public class TestFairyFeedbackOverlay {
+
+	public enum OverlayPurpose {
+		SCREENSHOT, VIDEO
+	}
+
+	/***************** Public Interface *****************/
+
+	/**
+	 * Call this before everything else to install a human friendly overlay UI
+	 * for sending feedbacks to TestFairy.
+	 *
+	 * @param context Application context or an Activity, must not be null.
+	 * @param appToken Your TestFairy app token.
+	 * @param showImmediately Set it to true of you want to show the overlay after installation completes.
+	 */
+	static public void installOverlay(Context context, String appToken, OverlayPurpose overlayPurpose, boolean showImmediately) {
```

Comment:
No need for showImmediately. If somebody calls this, then they want it visible right now.

Reply:
Good to know.